src/openshift/cluster.ts

```diff
@@ -61,7 +61,9 @@ export class Cluster extends OpenShiftItem {
     }
 
     static async getUrl(): Promise<string | null> {
+        const clusterURl = await Cluster.getUrlFromClipboard();
```

Comment:
You should use VSCode API for reading data from clipboard https://code.visualstudio.com/api/references/vscode-api#Clipboard..

Reply:
I am using VSCode API for reading data from clipboard.